homeassistant/components/gogogate2/common.py

```diff
@@ -0,0 +1,133 @@
+"""Common code for GogoGate2 component."""
+from datetime import datetime, timedelta
+import logging
+from typing import Callable, NamedTuple, Optional
+
+from gogogate2_api import GogoGate2Api, InfoResponse
+from gogogate2_api.common import Door, get_configured_doors
+
+from homeassistant.components.gogogate2.const import (
```

Comment:
```suggestion
from .const import (
```

Please use relative imports so this can be tested in `custom_components`